src/modules/integrator-guide/pages/corteza-studio/index.adoc

```diff
@@ -0,0 +1,122 @@
+include::ROOT:partial$variables.adoc[]
+
+= {PRODUCT_NAME} Studio
+:description: {PRODUCT_NAME} Studio allows users to customize {PRODUCT_NAME} web applications styling according to their preferences.
+
+{PRODUCT_NAME} Studio allows users to customize {PRODUCT_NAME} web applications styling according to their preferences. 
+
+[IMPORTANT]
+====
+For {PRODUCT_NAME} Studio, to function properly, you need to xref:devops-guide:installing-sass/index.adoc[Install dart-sass binaries] in your system and 
+`HTTP_WEBAPP_ENABLED` enviroment variable set to `true`.
+====
+
+== Customizing {PRODUCT_NAME} Web Applications
+
+There are two ways of customizing your {PRODUCT_NAME} web applications style:
+
+=== Branding
+
+Under menu:User Interface[Settings] branding section.
```

Comment:
For these, I'd mention that it's on the admin webapp; something like

```suggestion
In the admin area, navigate to the menu:User Interface[Settings] branding section.
```

Reply:
This would apply to all similar occurrences (the one below this section for example)